starknet_py/tests/e2e/mock/contracts/universal_deployer.cairo

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts for Cairo v0.4.0 (utils/presets/UniversalDeployer.cairo)
+
```

Comment:
I think we should paste here whole oz's license text like I've done it [here](https://github.com/software-mansion/starknet.py/blob/update/compiled-contracts/starknet_py/tests/e2e/mock/contracts/oz_proxy.cairo).

Reply:
why not